mcs_benchmark_data/path.py

```diff
@@ -2,4 +2,4 @@
 
 ROOT_DIR_PATH = Path(__file__).parent.parent.absolute()
 
-DATA_DIR_PATH = ROOT_DIR_PATH / "data"
+DATA_DIR_PATH = ROOT_DIR_PATH / "test_data"
```

Comment:
This is now incorrect for real runs of the pipelines. Create a separate TEST_DATA_DIR_PATH constant and allow the pipeline (+ forward to pipeline phases) to take a data_dir_path: Path = DATA_DIR_PATH.

Reply:
Yes that is just meant to be a temporary work-around. I was trying to exercise restraint by not changing too too much.